rlbottraining/common_exercises/dribbling.py

```diff
@@ -17,13 +18,24 @@ class Dribbling(StrikerExercise):
     def make_game_state(self, rng: SeededRandomNumberGenerator) -> GameState:
         car_pos = Vector3(3500 * rng.n11(), rng.uniform(0, -4000), 25)
         ball_pos = Vector3(car_pos.x, car_pos.y + 500, 500)
-        ball_state = BallState(Physics(location=ball_pos, velocity=Vector3(0, 0, 500)))
-        car_state = CarState(boost_amount=87, jumped=True, double_jumped=True,
-                             physics=Physics(location=car_pos, velocity=Vector3(0, 0, 0),
-                                             rotation=Rotator(0, pi / 2, 0)))
-        enemy_car = CarState(physics=Physics(location=Vector3(10000, 10000, 10000)))
-        game_state = GameState(ball=ball_state, cars={0: car_state, 1: enemy_car})
-        return game_state
+        return GameState(
+            ball=BallState(physics=Physics(
+                location=ball_pos,
+                velocity=Vector3(0, 0, 500),
+                angular_velocity=Vector3(0, 0, 0))),
+            cars={
+                0: CarState(
+                    physics=Physics(
+                        location=car_pos,
+                        rotation=Rotator(0, pi / 2, 0),
+                        velocity=Vector3(0, 0, 0),
+                        angular_velocity=Vector3(0, 0, 0)),
+                    boost_amount=87),
+                1: CarState(physics=Physics(location=Vector3(10000, 10000, 10000)))
+            },
+            boosts={i: BoostState(0) for i in range(34)},
```

Comment:
This sadly doesn't do anything at the moment, might want to remove these to avoid confusion.

Reply:
Fixed this in my latest commit for both these exercises as well as other existing exercises.

Reply:
Thanks for fixing! It's merged 👍